pipeline/coordinator.go

```diff
@@ -385,7 +385,7 @@ func checkDisplayAspectRatio(track video.InputTrack, requestID string) bool {
 	resRatio := float64(track.Width) / float64(track.Height)
 
 	// calculate the difference between the aspect ratio and the real ratio of the resolution, allow up to 20%
-	diff := dar - resRatio
+	diff := math.Abs(dar - resRatio)
```

Comment:
is 20% difference what ffmpeg allows here or arbitrary?

Reply:
Arbitrary from me looking at 1000 recent uploads: https://github.com/livepeer/catalyst-api/pull/734#discussion_r1243472648 look ok to you?